static/skywire-manager-src/src/app/components/layout/top-bar/top-bar.component.ts

```diff
@@ -136,12 +206,104 @@ export class TopBarComponent implements OnInit, OnDestroy {
         this.hideLanguageButton = true;
       }
     }));
+
+    // Check if the app is being accessed from a remote localtion.
+    const currentHost = window.location.hostname;
+    if (!currentHost.toLowerCase().includes('localhost') && !currentHost.toLowerCase().includes('127.0.0.1')) {
+      this.remoteAccess = true;
+    }
   }
 
   ngOnDestroy() {
     this.langSubscriptionsGroup.forEach(sub => sub.unsubscribe());
     this.refreshRequested.complete();
     this.optionSelected.complete();
+    this.stopGettingVpnInfo();
+  }
+
+  // Start getting and showing the vpn info.
+  startGettingVpnInfo() {
+    this.showVpnInfo = true;
+    this.vpnClientService.initialize(this.localVpnKeyInternal);
+
+    // Update the data units.
+    this.updateVpnDataStatsUnit();
+
+    // Get the data.
+    this.vpnDataSubscription = this.vpnClientService.backendState.subscribe(data => {
+      if (data) {
+        this.vpnData = {
+          state: '',
+          stateClass: '',
+          latency: data.vpnClientAppData.connectionData ? data.vpnClientAppData.connectionData.latency : 0,
+          downloadSpeed: data.vpnClientAppData.connectionData ? data.vpnClientAppData.connectionData.downloadSpeed : 0,
+          uploadSpeed: data.vpnClientAppData.connectionData ? data.vpnClientAppData.connectionData.uploadSpeed : 0,
+        };
+
+        // Use the correct state vars.
+        if (data.vpnClientAppData.appState === AppState.Stopped) {
+          this.vpnData.state = 'vpn.connection-info.state-disconnected';
+          this.vpnData.stateClass = 'red-clear-text';
+        } else if (data.vpnClientAppData.appState === AppState.Connecting) {
+          this.vpnData.state = 'vpn.connection-info.state-connecting';
+          this.vpnData.stateClass = 'yellow-clear-text';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
          this.vpnData.stateClass = "yellow-clear-text";
```